src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RequestJudgementByAdmissionForSpecCuiCallbackHandler.java

```diff
@@ -64,7 +67,9 @@ private CallbackResponse validateDefaultJudgementEligibility(CallbackParams call
         CaseData.CaseDataBuilder<?, ?> caseDataBuilder = caseData.toBuilder();
         ArrayList<String> errors = new ArrayList<>();
         if (caseData.isJudgementDateNotPermitted()) {
-            errors.add(format(NOT_VALID_DJ_BY_ADMISSION, caseData.setUpJudgementFormattedPermittedDate()));
+            LocalDate extendedRespondent1ResponseDate = deadlineCalculatorService.calculateExtendedDeadline(
```

Comment:
`caseData.getRespondent1ResponseDate().toLocalDate().plusDays(5)`
Could you put that snake in separate variable please